src/Core/Infrastructure/StringHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
We have similar helpers in almost all our modules so that we can extract it into utils module, but not in this task. I register a technical debt.

Reply:
https://spryker.atlassian.net/browse/SDK-4713